sklearn/utils/metaestimators.py

```diff
@@ -189,7 +195,7 @@ def _safe_split(estimator, X, y, indices, train_indices=None):
         Indexed targets.
 
     """
-    if getattr(estimator, "_pairwise", False):
+    if _is_pairwise(estimator):
```

Comment:
This here is the main usage right? I think it might be nice to have a direct test of this? Or of ``cross_validate`` or anything like that? Right now you're only testing the helper (extensively) but you're not testing anything that actually uses the helper, right?

It might be a bit overkill to test all of these places but maybe one?

Reply:
This was done in https://github.com/scikit-learn/scikit-learn/pull/18143/commits/cfeaa0906d9939ac5a8a5c0c6305038cb3db9261